pep/builtin_client.go

```diff
@@ -0,0 +1,42 @@
+package pep
+
+import (
+	"golang.org/x/net/context"
+
+	ps "github.com/infobloxopen/themis/pdpserver/server"
+)
+
+type builtinClient struct {
+	s *ps.Server
+}
+
+func NewBuiltinClient(policyFile string, contentFiles []string) *builtinClient {
```

Comment:
Instead of making separate `NewBuiltinClient`. You can provide option `WithBuiltinServer`. So client can be created in common way - `NewClient(WithBuiltinServer(s))`